src/components/gravid/GravidReducer.ts

```diff
@@ -94,6 +87,10 @@ const GravidReducer = (
       nextState.kvittering = payload?.kvittering;
       return validateGravid(nextState);
 
+    case Actions.CloseServerFeilModal:
+      nextState.showModal = false;
```

Comment:
Synes denne burde hete ServerFeil som i koden ovenfor for å skape likhet mellom state og Action

Reply:
Verdien burde stilles tilbake til default verdien til state for ikke få mange "veier" i koden. Er false default verdi?